smartmin/users/views.py

```diff
@@ -303,6 +303,8 @@ def form_valid(self, form):
             no_user_email_template = getattr(settings, "NO_USER_FORGET_EMAIL_TEMPLATE",
                                              "smartmin/users/no_user_email.txt")
 
+            no_user_send_email = getattr(settings, "NO_USER_FOUND_SEND_EMAIL", False)
```

Comment:
why does this need to be a setting? seems like we should never send emails to addresses that aren't real users

Reply:
Yes, we default to "Not sending" but in the case where that makes sense to send them allowing that configuration seems fine to me

Reply:
Yeah I just think we should be conservative about adding new Django settings, and in this case it's _never_ the right behavior to send a forgotten password link to an address which isn't in the system.

Reply:
Updated